src/pages/api/schema/[version].openapi.json.ts

```diff
@@ -0,0 +1,53 @@
+import type { APIRoute } from "astro";
+
+import { getEveryEndpoints } from "~/layouts/rest-api/schema-utils/endpoint";
+import {
+  getOperation,
+  isQueryOrBodyOperation,
+} from "~/layouts/rest-api/schema-utils/operation";
+
+import v1Schema from "../../../schema/v1.openapi.json";
+import v2Schema from "../../../schema/v2.openapi.json";
+
+export const prerender = true;
+
+export function getStaticPaths() {
+  return [{ params: { version: "v1" } }, { params: { version: "v2" } }];
+}
+
+const schemaMap = new Map(
+  Object.entries({
+    v1: JSON.stringify(transformSchema(v1Schema)),
+    v2: JSON.stringify(transformSchema(v2Schema)),
+  }),
+);
+
+export const GET: APIRoute = ({ params, redirect }) => {
+  const { version } = params;
+  const schema = schemaMap.get(version ?? "");
+  if (!schema) return redirect("/404");
```

Comment:
```suggestion
  if (!schema) return new Response(null, { status: 404 })
```
이렇게 해 둬야 나중에 404 일괄적으로 처리하기 편할 것 같아요

Reply:
아 다른것들도 이렇게 404 처리하고 있군요